src/main/telemetry/telemetry.h

```diff
@@ -59,5 +59,4 @@ bool telemetryDetermineEnabledState(portSharing_e portSharing);
 
 void telemetryUseConfig(telemetryConfig_t *telemetryConfig);
 
-#define TELEMETRY_SHAREABLE_PORT_FUNCTIONS_MASK (FUNCTION_TELEMETRY_FRSKY | FUNCTION_TELEMETRY_LTM)
-
+#define TELEMETRY_SHAREABLE_PORT_FUNCTIONS_MASK (FUNCTION_TELEMETRY_FRSKY | FUNCTION_TELEMETRY_LTM | FUNCTION_TELEMETRY_CRSF)
```

Comment:
I mean can you for example combine SBUS serial RX and CSRF telemetry?

`TELEMETRY_SHAREABLE_PORT_FUNCTIONS_MASK` is used to specify the telemetry protocols that can be used in combination with any of the serial Rx protocols that do not require bidirectional communication.

Other protocols `JETIEXBUS` that tightly integrate serialRx and telemetry don't use `TELEMETRY_SHAREABLE_PORT_FUNCTIONS_MASK`, but enable telemetry whenever the corresponding serialRx protocol is selected.